libursa/src/encoding/hex.rs

```diff
@@ -2,27 +2,32 @@ use CryptoError;
 
 pub fn bin2hex(b: &[u8]) -> String {
     b.iter()
-     .map(|b| format!("{:02x}", b))
-     .collect::<Vec<_>>()
-     .join("")
+        .map(|b| format!("{:02x}", b))
```

Comment:
These look like tabs to me. We don't use tabs for spacing.

Reply:
I double checked. rustfmt uses spaces instead of tabs.